packages/matchbox/src/components/Slider/Slider.module.scss

```diff
@@ -62,3 +62,38 @@
     border: 2px solid color(gray, 4);
   }
 }
+
+.Tick {
+  position: absolute;
+  height: 10px;
+  width: 10px;
+  border-radius: 50%;
+  transform: translate(-3px, -3px);
+
```

Comment:
Curious, why the empty lines?

Reply:
I tend to separate box model, type, animation, interaction styles with a space like this